### DISCUSSION TITLE
Creating Audio Visual Content

### DISCUSSION THREAD

**Discussion by BekahHW** [Virtual-Coffee/virtualcoffee.io#526] [General Resources]
Do you have resources for creating AV content? We'd love to gather resources on video, podcasting, livestreaming, presentation creation.  We'd also love to have some of your favorite examples, so drop those below too!

**Comment by teezzan** [comment]
[Loom](https://www.loom.com/) is a great application for recording. [Miro](Miro.com) helps to make very cool images and diagrams. 

My favourite AV content is the[ Azure course videos](https://docs.microsoft.com/en-us/learn/modules/intro-to-azure-virtual-machines/2-compile-a-checklist-for-creating-a-vm?source=learn#) on Microsoft Learn because of their cool moving diagrams. I hope to be able to create such a masterpiece of an AV content.

**Comment by chrismjohnston** [comment]
1. [Screenflow](https://www.telestream.net/screenflow/overview.htm)  for screen recording.
2. [Blackhole,](https://github.com/ExistentialAudio/BlackHole) [Loopback,](https://rogueamoeba.com/loopback/) and [Audio Hijack Pro](https://rogueamoeba.com/audiohijackpro/) for rerouting sound sources.
3. Some cheap lights on amazon. If you are only going to have one light, you need a ring light. If you are going to have more than one, you need three. Google ["three light setup for videos"](https://lmgtfy.app/?q=three+light+setup+for+video) to see why.
4. And do not skimp on a microphone. There are great USB options but if you want the highest quality you need an XLR mic. XLR requires an audio interface device, so if you go down that road it can be expensive. A great alternative is the [Shure SM7B](https://www.sweetwater.com/store/detail/SM7B--shure-sm7b-cardioid-dynamic-vocal-microphone), an excellent UB mic but about $400.
5. There are some excellent courses on shooting for video on LinkedIn Learning and my top 3 YouTube video shooting channels are [Peter McKinnon,](https://www.youtube.com/user/petermckinnon24) [Indy Mogul,](https://www.youtube.com/user/indymogul) and [Film Riot](https://www.youtube.com/user/filmriot).

**Comment by truongmkevin** [comment]
### Video / Screencasting Software

- **[Loom](https://www.loom.com/)**: Fantastic option, comes with basic editing tools like trimming and moving, let's you record your screen while also having a small image of you / your webcam on the video, plus uploads directly to the cloud (though I have had instances happen 2% of the time where it didn't upload correctly and caused for re-recording 20 minute long videos... hopefully this is fixed) Free tier allows up to 5 minute videos, 25 videos max. Very good option even with their paid tier, which comes with no time limit and unlimited storage.
- **[Screencast-o-Matic](https://screencast-o-matic.com/ )**: is my current tool because at the time Loom did a price switcheroo on me and I didn't like that, ended up getting this for ~$3 a month. It's just as good as Loom feature wise, has extra features I don't use much, but UI/UX isn't as good as Loom's. Still, I love it and it gets the job done very well. Unlimited cloud storage, no time limit.
- **[Open Broadcaster Software AKA OBS](https://obsproject.com/)** - Free, open source, great tool, allows for HIGH customization for advanced videos (but nothing 90%+ of us will need), I've used it a lot in the past. However, there is some technical config to get things "right" and if you plan on editing, you'll need separate software for that. Not my first pick if you are looking to get up and going within a couple hours. No cloud upload / storage either. But because of it's popularity and the fact that every streamer uses it, there are a lot of tutorials on YouTube.

If you plan on paying, you cannot go wrong with either **Loom** or **Screencast-o-Matic**.

### Audio
Only one option I need to recommend:

- **[Audacity](https://www.audacityteam.org/)**: free, open source, not the best looking UI but it JUST WORKS. I've used this for all my audio recording / podcasting. Allows for all the editing you would need, because of it's popularity, there are a lot of YouTube videos showing how to use it.

Podcast Related
- **[Zencastr](https://zencastr.com/)** - Introduced to me by Virtual Coffee podcast when I was a guest! Really awesome tool for having guests and needing video to connect. If you are just recording yourself, stick with Audacity (above)
- **Hosting: [Anchor by Spotify](https://anchor.fm/)** - FREE so it's great for just starting out if you are experimenting and not sure you want to commit to a paid plan
- **Hosting: [Buzzsprout](https://www.buzzsprout.com)**- Another popular host for your podcast audio and gives you a player
- **Read this: [https://www.smartpassiveincome.com/guide/how-to-start-a-podcast-tutorial-pat-flynn/](https://www.smartpassiveincome.com/guide/how-to-start-a-podcast-tutorial-pat-flynn/)**

### Hardware
As Chris mentioned above, high quality inputs are important. For microphones, I also suggest a USB one, I use the Shure AT2005USB because if I decide to upgrade to XLR connectors (what the "pros" use), then I have that option.

As for video, a good webcam is important, but not everyone is ready to drop $75 on a webcam. But we can use our phone! I've been meaning to try this out but you can connect your phone to your computer and use it as a webcam [https://reincubate.com/camo/](https://reincubate.com/camo/)

**Comment by BekahHW** [comment]
Via @nickytonline in slack:

https://whitep4nth3r.com/blog/twitch-live-coding-setup-obs/

**Reply by nickytonline** [reply]
She's a streaming pro. 👏🏻

**Comment by BekahHW** [comment]
@nickytonline's [Lunch & Learn on Getting Started with Streaming](https://www.iamdeveloper.com/pages/talks/#heading-getting-started-with-streaming-on-twitch-nick-taylor-lunch-and-learn)
